### PR TITLE
build-source: reorganize to emphasize only on CiV and CiC

### PR DESCRIPTION
**Pull request by ttzeng**
Separate sections for CiV and CIC build instructions.

Signed-off-by: Tonny Tzeng <tonny.tzeng@intel.com>
Reviewed-by: Balan, Swaroop <swaroop.balan@intel.com>
Reviewed-by: Badrappan, Jeevaka <jeevaka.badrappan@intel.com>

### CODE REVIEW

**Comment by ttzeng** [comment]
@swaroopbalan PTAL on the requested editing.

**Comment by swaroopbalan** [comment]
Please change Build the OS image >> Build the CAAS image

**Comment by swaroopbalan** [comment]
Build the OS image >> Build the CAAS image

File: source/getting_started/build-source.rst [hunk 1]
```diff
@@ -149,53 +142,90 @@ Build the OS image
...
-    
-   .. note::
-      The *-j $(nproc)* argument instructs the builder to compile the source
-      code with parallel tasks. The generated kernelflinger executables
-      .ZIP file
-      (:file:`out/target/product/celadon/celadon.flashfiles.eng.${USER}.zip`)
-      is available after the build. You can follow :ref:`install-on-nuc` of
-      this guide to flash the installer image to a removable USB drive and
-      install |C| on a |NUC|.
+       $ lunch cic-dev
```
Comment (JeevakaPrabu) [inline source/getting_started/build-source.rst]: cic-dev -> cic_dev-userdebug

Comment (ttzeng) [reply source/getting_started/build-source.rst]: @swaroopbalan @JeevakaPrabu Thanks for feedbacks, they are incorporated. If you are all OK with the current commit, it should be published on 01.org after 8AM PST.

Comment (swaroopbalan) [reply source/getting_started/build-source.rst]: Yes , i am OK

**Pull request merged**